fixtures/dummy/incidents.go

```diff
@@ -24,32 +24,51 @@ var FirstComment = models.Comment{
 	CreatedBy:  JohnWick.ID,
 	Comment:    "This is a comment",
 	IncidentID: LogisticsAPIDownIncident.ID,
-	CreatedAt:  time.Now(),
-	UpdatedAt:  time.Now(),
+	CreatedAt:  time.Now().Add(-time.Hour),
```

Comment:
Why are we doing this ?

Reply:
The test fails sometimes and I think it's because of the unstable ordering of the comments, Although I am not able to reproduce it enough times to make sure of it. I tried a few different ways to rewrite the SQL query to order the comment but the query was too verbose.

I was just experimenting with time to see if it helps in stable ordering.

Reply:
I have modified the query